[package] sdl/2.0.16: Failed to compile

<!-- 
  Please don't forget to update the issue title.
  Include all applicable information to help us reproduce your problem.
-->

### Package and Environment Details (include every applicable attribute)
  * Package Name/Version: **sdl/2.0.16**
  * Operating System+version: **archlinux 2021-08-29**
  * Compiler+version: **GCC 11.1**
  * Conan version: **conan 1.39.0**
  * Python version: **Python 3.9.6**


### Conan profile
```
[settings]
os=Linux
os_build=Linux
arch=x86_64
arch_build=x86_64
compiler=gcc
compiler.version=11.1
compiler.libcxx=libstdc++11
build_type=Release
[options]
[conf]
[build_requires]
[env]
```


### Steps to reproduce
```bash
conan install sdl/2.0.16@ --build=missing
```

### Logs
<details><summary>Click to expand log</summary>

```
Configuration:
[settings]
arch=x86_64
arch_build=x86_64
build_type=Release
compiler=gcc
compiler.libcxx=libstdc++11
compiler.version=11.1
os=Linux
os_build=Linux
[options]
[build_requires]
[env]

sdl/2.0.16: WARN: Package binary is corrupted, removing: 7be1ce258372c81cde134e98e4254a059071ec45
Installing package: sdl/2.0.16
Requirements
    egl/system from 'conancenter' - Cache
    expat/2.4.1 from 'conancenter' - Cache
    flac/1.3.3 from 'conancenter' - Cache
    libalsa/1.2.4 from 'conancenter' - Cache
    libcap/2.50 from 'conancenter' - Cache
    libffi/3.4.2 from 'conancenter' - Cache
    libiconv/1.16 from 'conancenter' - Cache
    libsndfile/1.0.31 from 'conancenter' - Cache
    libxml2/2.9.12 from 'conancenter' - Cache
    nas/1.9.4 from 'conancenter' - Cache
    ogg/1.3.4 from 'conancenter' - Cache
    opengl/system from 'conancenter' - Cache
    openssl/1.1.1k from 'conancenter' - Cache
    opus/1.3.1 from 'conancenter' - Cache
    pulseaudio/14.2 from 'conancenter' - Cache
    sdl/2.0.16 from 'conancenter' - Cache
    vorbis/1.3.7 from 'conancenter' - Cache
    wayland/1.19.0 from 'conancenter' - Cache
    wayland-protocols/1.21 from 'conancenter' - Cache
    xkbcommon/1.3.0 from 'conancenter' - Cache
    xorg/system from 'conancenter' - Cache
    zlib/1.2.11 from 'conancenter' - Cache
Packages
    egl/system:5ab84d6acfe1f23c4fae0ab88f26e3a396351ac9 - Cache
    expat/2.4.1:9152488a2cb1b9f84aab0a3d1bf3e8baccc7fbac - Cache
    flac/1.3.3:c0ee7360ba61fdb5568cc4c1bcb0ebd49e4aa293 - Cache
    libalsa/1.2.4:c9856c71dedf45b7eef47b38ef5a5fca622fe1d1 - Cache
    libcap/2.50:58aa52996642505eee5f4042cedcf9b4d1c57eb4 - Cache
    libffi/3.4.2:58aa52996642505eee5f4042cedcf9b4d1c57eb4 - Cache
    libiconv/1.16:58aa52996642505eee5f4042cedcf9b4d1c57eb4 - Cache
    libsndfile/1.0.31:6d4901686f6e21208e189e4a8256c909777ff39d - Cache
    libxml2/2.9.12:31ecb4cd2df6ef54df58f0d2e00279c695bf5896 - Cache
    nas/1.9.4:077884758292d626fd4366e95bb9fe5d8f397c2e - Cache
    ogg/1.3.4:58aa52996642505eee5f4042cedcf9b4d1c57eb4 - Cache
    opengl/system:5ab84d6acfe1f23c4fae0ab88f26e3a396351ac9 - Cache
    openssl/1.1.1k:58aa52996642505eee5f4042cedcf9b4d1c57eb4 - Cache
    opus/1.3.1:58aa52996642505eee5f4042cedcf9b4d1c57eb4 - Cache
    pulseaudio/14.2:1652e103f03ba70f8404bd7d95cb5314c18ac560 - Cache
    sdl/2.0.16:7be1ce258372c81cde134e98e4254a059071ec45 - Build
    vorbis/1.3.7:92bf7eef3b44b5628cca3f9dd5c3cf74e6740617 - Cache
    wayland/1.19.0:b692d8f5246cdff4da1af4d2015ce9e023b13b6b - Cache
    wayland-protocols/1.21:5ab84d6acfe1f23c4fae0ab88f26e3a396351ac9 - Cache
    xkbcommon/1.3.0:d6009e6c2a16e061040d090b0136893091c04ae1 - Cache
    xorg/system:5ab84d6acfe1f23c4fae0ab88f26e3a396351ac9 - Cache
    zlib/1.2.11:58aa52996642505eee5f4042cedcf9b4d1c57eb4 - Cache
Build requirements
    pkgconf/1.7.4 from 'conancenter' - Cache
Build requirements packages
    pkgconf/1.7.4:58aa52996642505eee5f4042cedcf9b4d1c57eb4 - Cache

Installing (downloading, building) binaries...
egl/system: Already installed!
expat/2.4.1: Already installed!
libalsa/1.2.4: Already installed!
libcap/2.50: Already installed!
libffi/3.4.2: Already installed!
libiconv/1.16: Already installed!
libiconv/1.16: Appending PATH environment var: /home/stella/.conan/data/libiconv/1.16/_/_/package/58aa52996642505eee5f4042cedcf9b4d1c57eb4/bin
ogg/1.3.4: Already installed!
opengl/system: Already installed!
openssl/1.1.1k: Already installed!
opus/1.3.1: Already installed!
pkgconf/1.7.4: Already installed!
pkgconf/1.7.4: Appending PATH env var: /home/stella/.conan/data/pkgconf/1.7.4/_/_/package/58aa52996642505eee5f4042cedcf9b4d1c57eb4/bin
pkgconf/1.7.4: Setting PKG_CONFIG env var: /home/stella/.conan/data/pkgconf/1.7.4/_/_/package/58aa52996642505eee5f4042cedcf9b4d1c57eb4/bin/pkgconf
pkgconf/1.7.4: Appending AUTOMAKE_CONAN_INCLUDES env var: /home/stella/.conan/data/pkgconf/1.7.4/_/_/package/58aa52996642505eee5f4042cedcf9b4d1c57eb4/bin/aclocal
wayland-protocols/1.21: Already installed!
xorg/system: Already installed!
zlib/1.2.11: Already installed!
flac/1.3.3: Already installed!
flac/1.3.3: Appending PATH environment variable: /home/stella/.conan/data/flac/1.3.3/_/_/package/c0ee7360ba61fdb5568cc4c1bcb0ebd49e4aa293/bin
libxml2/2.9.12: Already installed!
libxml2/2.9.12: Appending PATH environment variable: /home/stella/.conan/data/libxml2/2.9.12/_/_/package/31ecb4cd2df6ef54df58f0d2e00279c695bf5896/bin
nas/1.9.4: Already installed!
nas/1.9.4: Appending PATH environment variable: /home/stella/.conan/data/nas/1.9.4/_/_/package/077884758292d626fd4366e95bb9fe5d8f397c2e/bin
vorbis/1.3.7: Already installed!
libsndfile/1.0.31: Already installed!
libsndfile/1.0.31: Appending PATH environment variable: /home/stella/.conan/data/libsndfile/1.0.31/_/_/package/6d4901686f6e21208e189e4a8256c909777ff39d/bin
wayland/1.19.0: Already installed!
wayland/1.19.0: Appending PATH environment variable: /home/stella/.conan/data/wayland/1.19.0/_/_/package/b692d8f5246cdff4da1af4d2015ce9e023b13b6b/bin
pulseaudio/14.2: Already installed!
xkbcommon/1.3.0: Already installed!
xkbcommon/1.3.0: Appending PATH environment variable: /home/stella/.conan/data/xkbcommon/1.3.0/_/_/package/d6009e6c2a16e061040d090b0136893091c04ae1/bin
sdl/2.0.16: Applying build-requirement: pkgconf/1.7.4
sdl/2.0.16: WARN: Build folder is dirty, removing it: /home/stella/.conan/data/sdl/2.0.16/_/_/build/7be1ce258372c81cde134e98e4254a059071ec45
sdl/2.0.16: Copying sources to build folder
sdl/2.0.16: Building your package in /home/stella/.conan/data/sdl/2.0.16/_/_/build/7be1ce258372c81cde134e98e4254a059071ec45
sdl/2.0.16: Generator cmake created conanbuildinfo.cmake
sdl/2.0.16: Generator pkg_config created libpkgconf.pc
sdl/2.0.16: Generator pkg_config created libpulse.pc
sdl/2.0.16: Generator pkg_config created libpulse-simple.pc
sdl/2.0.16: Generator pkg_config created pulseaudio.pc
sdl/2.0.16: Generator pkg_config created opengl.pc
sdl/2.0.16: Generator pkg_config created nas.pc
sdl/2.0.16: Generator pkg_config created xkbcli-interactive-wayland.pc
sdl/2.0.16: Generator pkg_config created xkbregistry.pc
sdl/2.0.16: Generator pkg_config created xkbcommon.pc
sdl/2.0.16: Generator pkg_config created xkbcommon-x11.pc
sdl/2.0.16: Generator pkg_config created xkbcommon_full_package.pc
sdl/2.0.16: Generator pkg_config created egl.pc
sdl/2.0.16: Generator pkg_config created alsa.pc
sdl/2.0.16: Generator pkg_config created sndfile.pc
sdl/2.0.16: Generator pkg_config created libcap.pc
sdl/2.0.16: Generator pkg_config created xcb-dri3.pc
sdl/2.0.16: Generator pkg_config created xcb-util.pc
sdl/2.0.16: Generator pkg_config created xcb-event.pc
sdl/2.0.16: Generator pkg_config created xcb-aux.pc
sdl/2.0.16: Generator pkg_config created xcb-atom.pc
sdl/2.0.16: Generator pkg_config created xkeyboard-config.pc
sdl/2.0.16: Generator pkg_config created xcb.pc
sdl/2.0.16: Generator pkg_config created xcb-xinerama.pc
sdl/2.0.16: Generator pkg_config created xcb-xfixes.pc
sdl/2.0.16: Generator pkg_config created xcb-sync.pc
sdl/2.0.16: Generator pkg_config created xcb-shm.pc
sdl/2.0.16: Generator pkg_config created xcb-shape.pc
sdl/2.0.16: Generator pkg_config created xcb-renderutil.pc
sdl/2.0.16: Generator pkg_config created xcb-render.pc
sdl/2.0.16: Generator pkg_config created xcb-randr.pc
sdl/2.0.16: Generator pkg_config created xcb-keysyms.pc
sdl/2.0.16: Generator pkg_config created xcb-image.pc
sdl/2.0.16: Generator pkg_config created xcb-icccm.pc
sdl/2.0.16: Generator pkg_config created xcb-xkb.pc
sdl/2.0.16: Generator pkg_config created xtrans.pc
sdl/2.0.16: Generator pkg_config created xxf86vm.pc
sdl/2.0.16: Generator pkg_config created xvmc.pc
sdl/2.0.16: Generator pkg_config created xv.pc
sdl/2.0.16: Generator pkg_config created xtst.pc
sdl/2.0.16: Generator pkg_config created xt.pc
sdl/2.0.16: Generator pkg_config created xscrnsaver.pc
sdl/2.0.16: Generator pkg_config created xres.pc
sdl/2.0.16: Generator pkg_config created xrender.pc
sdl/2.0.16: Generator pkg_config created xrandr.pc
sdl/2.0.16: Generator pkg_config created xpm.pc
sdl/2.0.16: Generator pkg_config created xmuu.pc
sdl/2.0.16: Generator pkg_config created xmu.pc
sdl/2.0.16: Generator pkg_config created xkbfile.pc
sdl/2.0.16: Generator pkg_config created xinerama.pc
sdl/2.0.16: Generator pkg_config created xi.pc
sdl/2.0.16: Generator pkg_config created xft.pc
sdl/2.0.16: Generator pkg_config created xfixes.pc
sdl/2.0.16: Generator pkg_config created xext.pc
sdl/2.0.16: Generator pkg_config created xdmcp.pc
sdl/2.0.16: Generator pkg_config created xdamage.pc
sdl/2.0.16: Generator pkg_config created xcursor.pc
sdl/2.0.16: Generator pkg_config created xcomposite.pc
sdl/2.0.16: Generator pkg_config created xaw7.pc
sdl/2.0.16: Generator pkg_config created xau.pc
sdl/2.0.16: Generator pkg_config created sm.pc
sdl/2.0.16: Generator pkg_config created ice.pc
sdl/2.0.16: Generator pkg_config created fontenc.pc
sdl/2.0.16: Generator pkg_config created x11-xcb.pc
sdl/2.0.16: Generator pkg_config created x11.pc
sdl/2.0.16: Generator pkg_config created xorg.pc
sdl/2.0.16: Generator pkg_config created libcrypto.pc
sdl/2.0.16: Generator pkg_config created libssl.pc
sdl/2.0.16: Generator pkg_config created openssl.pc
sdl/2.0.16: Generator pkg_config created wayland-egl-backend.pc
sdl/2.0.16: Generator pkg_config created wayland-client.pc
sdl/2.0.16: Generator pkg_config created wayland-egl.pc
sdl/2.0.16: Generator pkg_config created wayland-cursor.pc
sdl/2.0.16: Generator pkg_config created wayland-server.pc
sdl/2.0.16: Generator pkg_config created wayland-scanner.pc
sdl/2.0.16: Generator pkg_config created wayland.pc
sdl/2.0.16: Generator pkg_config created wayland-protocols.pc
sdl/2.0.16: Generator pkg_config created vorbis.pc
sdl/2.0.16: Generator pkg_config created vorbisfile.pc
sdl/2.0.16: Generator pkg_config created vorbisfile-alias.pc
sdl/2.0.16: Generator pkg_config created vorbisenc.pc
sdl/2.0.16: Generator pkg_config created vorbisenc-alias.pc
sdl/2.0.16: Generator pkg_config created vorbis_full_package.pc
sdl/2.0.16: Generator pkg_config created flac.pc
sdl/2.0.16: Generator pkg_config created flac++.pc
sdl/2.0.16: Generator pkg_config created opus.pc
sdl/2.0.16: Generator pkg_config created libffi.pc
sdl/2.0.16: Generator pkg_config created libxml-2.0.pc
sdl/2.0.16: Generator pkg_config created expat.pc
sdl/2.0.16: Generator pkg_config created libiconv.pc
sdl/2.0.16: Generator pkg_config created ogglib.pc
sdl/2.0.16: Generator pkg_config created ogg.pc
sdl/2.0.16: Generator pkg_config created zlib.pc
sdl/2.0.16: Aggregating env generators
sdl/2.0.16: Calling build()
-- The C compiler identification is GNU 11.1.0
-- The CXX compiler identification is GNU 11.1.0
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /usr/bin/cc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /usr/bin/c++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Conan: called by CMake conan helper
-- Conan: called inside local cache
-- Conan: Adjusting output directories
-- Conan: Using cmake global configuration
-- Conan: Adjusting default RPATHs Conan policies
-- Conan: Adjusting language standard
-- Conan: Adjusting fPIC flag (ON)
-- Conan: Compiler GCC>=5, checking major version 11.1
-- Conan: Checking correct version: 11.1
-- Found PkgConfig: /home/stella/.conan/data/pkgconf/1.7.4/_/_/package/58aa52996642505eee5f4042cedcf9b4d1c57eb4/bin/pkgconf (found version "1.7.4")
-- Performing Test HAVE_GCC_WALL
-- Performing Test HAVE_GCC_WALL - Success
-- Performing Test HAVE_GCC_NO_STRICT_ALIASING
-- Performing Test HAVE_GCC_NO_STRICT_ALIASING - Success
-- Performing Test HAVE_GCC_WDECLARATION_AFTER_STATEMENT
-- Performing Test HAVE_GCC_WDECLARATION_AFTER_STATEMENT - Success
-- Performing Test HAVE_GCC_WERROR_DECLARATION_AFTER_STATEMENT
-- Performing Test HAVE_GCC_WERROR_DECLARATION_AFTER_STATEMENT - Success
-- Performing Test HAVE_GCC_ATOMICS
-- Performing Test HAVE_GCC_ATOMICS - Success
-- Performing Test HAVE_GCC_PREFERRED_STACK_BOUNDARY
-- Performing Test HAVE_GCC_PREFERRED_STACK_BOUNDARY - Failed
-- Performing Test HAVE_GCC_FVISIBILITY
-- Performing Test HAVE_GCC_FVISIBILITY - Success
-- Performing Test HAVE_GCC_WSHADOW
-- Performing Test HAVE_GCC_WSHADOW - Success
-- Performing Test HAVE_NO_UNDEFINED
-- Performing Test HAVE_NO_UNDEFINED - Success
-- Performing Test HAVE_MMX
-- Performing Test HAVE_MMX - Success
-- Performing Test HAVE_3DNOW
-- Performing Test HAVE_3DNOW - Success
-- Performing Test HAVE_SSE
-- Performing Test HAVE_SSE - Success
-- Performing Test HAVE_SSE2
-- Performing Test HAVE_SSE2 - Success
-- Performing Test HAVE_SSE3
-- Performing Test HAVE_SSE3 - Success
-- Looking for immintrin.h
-- Looking for immintrin.h - found
-- Performing Test HAVE_ALTIVEC_H_HDR
-- Performing Test HAVE_ALTIVEC_H_HDR - Failed
-- Performing Test HAVE_ALTIVEC
-- Performing Test HAVE_ALTIVEC - Failed
-- Looking for sys/types.h
-- Looking for sys/types.h - found
-- Looking for stdio.h
-- Looking for stdio.h - found
-- Looking for stdlib.h
-- Looking for stdlib.h - found
-- Looking for stddef.h
-- Looking for stddef.h - found
-- Looking for stdarg.h
-- Looking for stdarg.h - found
-- Looking for malloc.h
-- Looking for malloc.h - found
-- Looking for memory.h
-- Looking for memory.h - found
-- Looking for string.h
-- Looking for string.h - found
-- Looking for limits.h
-- Looking for limits.h - found
-- Looking for strings.h
-- Looking for strings.h - found
-- Looking for wchar.h
-- Looking for wchar.h - found
-- Looking for inttypes.h
-- Looking for inttypes.h - found
-- Looking for stdint.h
-- Looking for stdint.h - found
-- Looking for ctype.h
-- Looking for ctype.h - found
-- Looking for math.h
-- Looking for math.h - found
-- Looking for iconv.h
-- Looking for iconv.h - found
-- Looking for signal.h
-- Looking for signal.h - found
-- Looking for libunwind.h
-- Looking for libunwind.h - found
-- Looking for 7 include files stddef.h, ..., float.h
-- Looking for 7 include files stddef.h, ..., float.h - found
-- Check size of size_t
-- Check size of size_t - done
-- Looking for M_PI
-- Looking for M_PI - found
-- Performing Test HAVE_MPROTECT
-- Performing Test HAVE_MPROTECT - Success
-- Looking for strtod
-- Looking for strtod - found
-- Looking for malloc
-- Looking for malloc - found
-- Looking for calloc
-- Looking for calloc - found
-- Looking for realloc
-- Looking for realloc - found
-- Looking for free
-- Looking for free - found
-- Looking for getenv
-- Looking for getenv - found
-- Looking for setenv
-- Looking for setenv - found
-- Looking for putenv
-- Looking for putenv - found
-- Looking for unsetenv
-- Looking for unsetenv - found
-- Looking for qsort
-- Looking for qsort - found
-- Looking for abs
-- Looking for abs - found
-- Looking for bcopy
-- Looking for bcopy - found
-- Looking for memset
-- Looking for memset - found
-- Looking for memcpy
-- Looking for memcpy - found
-- Looking for memmove
-- Looking for memmove - found
-- Looking for memcmp
-- Looking for memcmp - found
-- Looking for strlen
-- Looking for strlen - found
-- Looking for strlcpy
-- Looking for strlcpy - not found
-- Looking for strlcat
-- Looking for strlcat - not found
-- Looking for _strrev
-- Looking for _strrev - not found
-- Looking for _strupr
-- Looking for _strupr - not found
-- Looking for _strlwr
-- Looking for _strlwr - not found
-- Looking for index
-- Looking for index - found
-- Looking for rindex
-- Looking for rindex - found
-- Looking for strchr
-- Looking for strchr - found
-- Looking for strrchr
-- Looking for strrchr - found
-- Looking for strstr
-- Looking for strstr - found
-- Looking for strtok_r
-- Looking for strtok_r - found
-- Looking for itoa
-- Looking for itoa - not found
-- Looking for _ltoa
-- Looking for _ltoa - not found
-- Looking for _uitoa
-- Looking for _uitoa - not found
-- Looking for _ultoa
-- Looking for _ultoa - not found
-- Looking for strtol
-- Looking for strtol - found
-- Looking for strtoul
-- Looking for strtoul - found
-- Looking for _i64toa
-- Looking for _i64toa - not found
-- Looking for _ui64toa
-- Looking for _ui64toa - not found
-- Looking for strtoll
-- Looking for strtoll - found
-- Looking for strtoull
-- Looking for strtoull - found
-- Looking for atoi
-- Looking for atoi - found
-- Looking for atof
-- Looking for atof - found
-- Looking for strcmp
-- Looking for strcmp - found
-- Looking for strncmp
-- Looking for strncmp - found
-- Looking for _stricmp
-- Looking for _stricmp - not found
-- Looking for strcasecmp
-- Looking for strcasecmp - found
-- Looking for _strnicmp
-- Looking for _strnicmp - not found
-- Looking for strncasecmp
-- Looking for strncasecmp - found
-- Looking for wcscmp
-- Looking for wcscmp - found
-- Looking for _wcsdup
-- Looking for _wcsdup - not found
-- Looking for wcsdup
-- Looking for wcsdup - found
-- Looking for wcslcat
-- Looking for wcslcat - not found
-- Looking for wcslcpy
-- Looking for wcslcpy - not found
-- Looking for wcslen
-- Looking for wcslen - found
-- Looking for wcsncmp
-- Looking for wcsncmp - found
-- Looking for wcsstr
-- Looking for wcsstr - found
-- Looking for wcscasecmp
-- Looking for wcscasecmp - found
-- Looking for _wcsicmp
-- Looking for _wcsicmp - not found
-- Looking for wcsncasecmp
-- Looking for wcsncasecmp - found
-- Looking for _wcsnicmp
-- Looking for _wcsnicmp - not found
-- Looking for sscanf
-- Looking for sscanf - found
-- Looking for vsscanf
-- Looking for vsscanf - found
-- Looking for vsnprintf
-- Looking for vsnprintf - found
-- Looking for fopen64
-- Looking for fopen64 - not found
-- Looking for fseeko
-- Looking for fseeko - found
-- Looking for fseeko64
-- Looking for fseeko64 - not found
-- Looking for _Exit
-- Looking for _Exit - found
-- Looking for sigaction
-- Looking for sigaction - found
-- Looking for setjmp
-- Looking for setjmp - found
-- Looking for nanosleep
-- Looking for nanosleep - found
-- Looking for sysconf
-- Looking for sysconf - found
-- Looking for sysctlbyname
-- Looking for sysctlbyname - not found
-- Looking for getauxval
-- Looking for getauxval - found
-- Looking for elf_aux_info
-- Looking for elf_aux_info - not found
-- Looking for poll
-- Looking for poll - found
-- Looking for pow in m
-- Looking for pow in m - found
-- Looking for atan
-- Looking for atan - found
-- Looking for atan2
-- Looking for atan2 - found
-- Looking for atanf
-- Looking for atanf - found
-- Looking for atan2f
-- Looking for atan2f - found
-- Looking for ceil
-- Looking for ceil - found
-- Looking for ceilf
-- Looking for ceilf - found
-- Looking for copysign
-- Looking for copysign - found
-- Looking for copysignf
-- Looking for copysignf - found
-- Looking for cos
-- Looking for cos - found
-- Looking for cosf
-- Looking for cosf - found
-- Looking for exp
-- Looking for exp - found
-- Looking for expf
-- Looking for expf - found
-- Looking for fabs
-- Looking for fabs - found
-- Looking for fabsf
-- Looking for fabsf - found
-- Looking for floor
-- Looking for floor - found
-- Looking for floorf
-- Looking for floorf - found
-- Looking for fmod
-- Looking for fmod - found
-- Looking for fmodf
-- Looking for fmodf - found
-- Looking for log
-- Looking for log - found
-- Looking for logf
-- Looking for logf - found
-- Looking for log10
-- Looking for log10 - found
-- Looking for log10f
-- Looking for log10f - found
-- Looking for lround
-- Looking for lround - found
-- Looking for lroundf
-- Looking for lroundf - found
-- Looking for pow
-- Looking for pow - found
-- Looking for powf
-- Looking for powf - found
-- Looking for round
-- Looking for round - found
-- Looking for roundf
-- Looking for roundf - found
-- Looking for scalbn
-- Looking for scalbn - found
-- Looking for scalbnf
-- Looking for scalbnf - found
-- Looking for sin
-- Looking for sin - found
-- Looking for sinf
-- Looking for sinf - found
-- Looking for sqrt
-- Looking for sqrt - found
-- Looking for sqrtf
-- Looking for sqrtf - found
-- Looking for tan
-- Looking for tan - found
-- Looking for tanf
-- Looking for tanf - found
-- Looking for acos
-- Looking for acos - found
-- Looking for acosf
-- Looking for acosf - found
-- Looking for asin
-- Looking for asin - found
-- Looking for asinf
-- Looking for asinf - found
-- Looking for trunc
-- Looking for trunc - found
-- Looking for truncf
-- Looking for truncf - found
-- Looking for iconv_open in iconv
-- Looking for iconv_open in iconv - found
-- Looking for alloca.h
-- Looking for alloca.h - found
-- Looking for alloca
-- Looking for alloca - found
-- Looking for alloca
-- Looking for alloca - found
-- Looking for alloca
-- Looking for alloca - not found
-- Performing Test HAVE_SA_SIGACTION
-- Performing Test HAVE_SA_SIGACTION - Success
-- Looking for dlopen
-- Looking for dlopen - not found
-- Looking for dlopen in dl
-- Looking for dlopen in dl - found
-- Performing Test OSS_FOUND
-- Performing Test OSS_FOUND - Success
-- dynamic libasound -> libasound.a
-- Checking for module 'libpipewire-0.3>=0.3.20'
--   Package 'libpipewire-0.3', required by 'virtual:world', not found
-- Checking for module 'libpulse-simple'
--   Found libpulse-simple, version 14.2
-- dynamic libpulse-simple -> libpulse-simple.a
-- Checking for module 'esound'
--   Package 'esound', required by 'virtual:world', not found
-- Looking for audio/audiolib.h
-- Looking for audio/audiolib.h - found
-- Looking for samplerate.h
-- Looking for samplerate.h - found
-- dynamic libsamplerate -> libsamplerate.so.0
-- dynamic libX11 -> libX11.so.6
-- dynamic libXext -> libXext.so.6
-- dynamic libXcursor -> libXcursor.so.1
-- dynamic libXinerama -> libXinerama.so.1
-- dynamic libXi -> libXi.so.6
-- dynamic libXrandr -> libXrandr.so.2
-- dynamic libXrender -> libXrender.so.1
-- dynamic libXss -> libXss.so.1
-- dynamic libXxf86vm -> libXxf86vm.so.1
-- Looking for X11/extensions/XInput2.h
-- Looking for X11/extensions/XInput2.h - found
-- Looking for X11/extensions/Xrender.h
-- Looking for X11/extensions/Xrender.h - found
-- Looking for shmat
-- Looking for shmat - found
-- Performing Test HAVE_CONST_XEXT_ADDDISPLAY
-- Performing Test HAVE_CONST_XEXT_ADDDISPLAY - Success
-- Performing Test HAVE_XGENERICEVENT
-- Performing Test HAVE_XGENERICEVENT - Success
-- Looking for XkbKeycodeToKeysym
-- Looking for XkbKeycodeToKeysym - found
-- Performing Test HAVE_XINPUT2_MULTITOUCH
-- Performing Test HAVE_XINPUT2_MULTITOUCH - Success
-- Checking for module 'egl'
--   Found egl, version system
-- Performing Test HAVE_VIDEO_OPENGL_EGL
-- Performing Test HAVE_VIDEO_OPENGL_EGL - Success
-- Checking for modules 'libdrm;gbm;egl'
--   Package 'libdrm', required by 'virtual:world', not found
--   Package 'gbm', required by 'virtual:world', not found
-- Performing Test HAVE_VIDEO_OPENGL_GLX
-- Performing Test HAVE_VIDEO_OPENGL_GLX - Success
-- Performing Test HAVE_VIDEO_OPENGL
-- Performing Test HAVE_VIDEO_OPENGL - Success
-- Performing Test HAVE_VIDEO_OPENGLES_V1
-- Performing Test HAVE_VIDEO_OPENGLES_V1 - Failed
-- Performing Test HAVE_VIDEO_OPENGLES_V2
-- Performing Test HAVE_VIDEO_OPENGLES_V2 - Success
-- Checking for modules 'wayland-client;wayland-egl;wayland-cursor;egl;xkbcommon'
--   Found wayland-client, version 1.19.0
--   Found wayland-egl, version 1.19.0
--   Found wayland-cursor, version 1.19.0
--   Found egl, version system
--   Found xkbcommon, version 1.3.0
Force WAYLAND_SCANNER to 'wayland-scanner', it's provided by Conan as build-requires
-- Checking for module 'libdecor-0'
--   Package 'libdecor-0', required by 'virtual:world', not found
-- Performing Test HAVE_VIDEO_VIVANTE_VDK
-- Performing Test HAVE_VIDEO_VIVANTE_VDK - Failed
-- Performing Test HAVE_VIDEO_VIVANTE_EGL_FB
-- Performing Test HAVE_VIDEO_VIVANTE_EGL_FB - Failed
-- Performing Test HAVE_INPUT_EVENTS
-- Performing Test HAVE_INPUT_EVENTS - Success
-- Performing Test HAVE_INPUT_KD
-- Performing Test HAVE_INPUT_KD - Success
-- Looking for libudev.h
-- Looking for libudev.h - found
-- Looking for sys/inotify.h
-- Looking for sys/inotify.h - found
-- Looking for inotify_init
-- Looking for inotify_init - found
-- Looking for inotify_init1
-- Looking for inotify_init1 - found
-- Checking for one of the modules 'dbus-1;dbus'
-- Checking for one of the modules 'ibus-1.0;ibus'
-- Checking for one of the modules 'libunwind'
CMake Error at /usr/share/cmake-3.21/Modules/FindPkgConfig.cmake:849 (message):
  None of the required 'libunwind' found
Call Stack (most recent call first):
  source_subfolder/CMakeLists.txt:1321 (pkg_search_module)


-- Checking for one of the modules 'libunwind-generic'
CMake Error at /usr/share/cmake-3.21/Modules/FindPkgConfig.cmake:849 (message):
  None of the required 'libunwind-generic' found
Call Stack (most recent call first):
  source_subfolder/CMakeLists.txt:1322 (pkg_search_module)


-- Performing Test HAVE_PTHREADS
-- Performing Test HAVE_PTHREADS - Success
-- Performing Test HAVE_RECURSIVE_MUTEXES
-- Performing Test HAVE_RECURSIVE_MUTEXES - Success
-- Performing Test HAVE_PTHREADS_SEM
-- Performing Test HAVE_PTHREADS_SEM - Success
-- Performing Test HAVE_SEM_TIMEDWAIT
-- Performing Test HAVE_SEM_TIMEDWAIT - Success
-- Looking for include file pthread.h
-- Looking for include file pthread.h - found
-- Looking for include file pthread_np.h
-- Looking for include file pthread_np.h - not found
-- Looking for pthread_setname_np
-- Looking for pthread_setname_np - not found
-- Looking for linux/version.h
-- Looking for linux/version.h - found
--
-- SDL2 was configured with the following options:
--
-- Platform: Linux-5.13.13-zen1-1-zen
-- 64-bit:   TRUE
-- Compiler: /usr/bin/cc
--
-- Subsystems:
--   Atomic:	ON
--   Audio:	ON
--   Video:	ON
--   Render:	ON
--   Events:	ON
--   Joystick:	ON
--   Haptic:	ON
--   Power:	ON
--   Threads:	ON
--   Timers:	ON
--   File:	ON
--   Loadso:	ON
--   CPUinfo:	ON
--   Filesystem:	ON
--   Dlopen:	ON
--   Sensor:	ON
--   Locale:	ON
--
-- Options:
--   3DNOW                  (Wanted: ON): ON
--   ALSA                   (Wanted: True): ON
--   ALSA_SHARED            (Wanted: ON): ON
--   ALTIVEC                (Wanted: ON): OFF
--   ARMNEON                (Wanted: OFF): OFF
--   ARMSIMD                (Wanted: OFF): OFF
--   ARTS                   (Wanted: ON): OFF
--   ARTS_SHARED            (Wanted: ON): OFF
--   ASAN                   (Wanted: OFF): OFF
--   ASSEMBLY               (Wanted: ON): ON
--   ASSERTIONS             (Wanted: auto): auto
--   BACKGROUNDING_SIGNAL   (Wanted: OFF): OFF
--   CLOCK_GETTIME          (Wanted: OFF): OFF
--   DIRECTFB_SHARED        (Wanted: OFF): OFF
--   DIRECTX                (Wanted: OFF): OFF
--   DISKAUDIO              (Wanted: ON): ON
--   DUMMYAUDIO             (Wanted: ON): ON
--   ESD                    (Wanted: ON): OFF
--   ESD_SHARED             (Wanted: ON): OFF
--   FOREGROUNDING_SIGNAL   (Wanted: OFF): OFF
--   FUSIONSOUND            (Wanted: OFF): OFF
--   FUSIONSOUND_SHARED     (Wanted: OFF): OFF
--   GCC_ATOMICS            (Wanted: ON): ON
--   HIDAPI                 (Wanted: OFF): OFF
--   JACK                   (Wanted: False): OFF
--   JACK_SHARED            (Wanted: OFF): OFF
--   JOYSTICK_VIRTUAL       (Wanted: ON): OFF
--   KMSDRM_SHARED          (Wanted: ON): OFF
--   LIBC                   (Wanted: ON): ON
--   LIBDECOR_SHARED        (Wanted: ON): OFF
--   LIBSAMPLERATE          (Wanted: ON): ON
--   LIBSAMPLERATE_SHARED   (Wanted: ON): ON
--   MMX                    (Wanted: ON): ON
--   NAS                    (Wanted: True): ON
--   NAS_SHARED             (Wanted: False): OFF
--   OSS                    (Wanted: ON): ON
--   PIPEWIRE               (Wanted: ON): OFF
--   PIPEWIRE_SHARED        (Wanted: ON): OFF
--   PTHREADS               (Wanted: ON): ON
--   PTHREADS_SEM           (Wanted: ON): ON
--   PULSEAUDIO             (Wanted: True): ON
--   PULSEAUDIO_SHARED      (Wanted: ON): ON
--   RENDER_D3D             (Wanted: OFF): OFF
--   RENDER_METAL           (Wanted: OFF): OFF
--   RPATH                  (Wanted: ON): OFF
--   SDL_DLOPEN             (Wanted: ON): ON
--   SDL_STATIC_PIC         (Wanted: True): OFF
--   SDL_TEST               (Wanted: OFF): OFF
--   SNDIO                  (Wanted: False): OFF
--   SNDIO_SHARED           (Wanted: ON): OFF
--   SSE                    (Wanted: ON): ON
--   SSE2                   (Wanted: ON): ON
--   SSE3                   (Wanted: ON): ON
--   SSEMATH                (Wanted: ON): OFF
--   VIDEO_COCOA            (Wanted: OFF): OFF
--   VIDEO_DIRECTFB         (Wanted: False): OFF
--   VIDEO_DUMMY            (Wanted: ON): ON
--   VIDEO_KMSDRM           (Wanted: ON): OFF
--   VIDEO_METAL            (Wanted: OFF): OFF
--   VIDEO_OFFSCREEN        (Wanted: OFF): OFF
--   VIDEO_OPENGL           (Wanted: True): ON
--   VIDEO_OPENGLES         (Wanted: True): ON
--   VIDEO_RPI              (Wanted: False): OFF
--   VIDEO_VIVANTE          (Wanted: ON): OFF
--   VIDEO_VULKAN           (Wanted: True): ON
--   VIDEO_WAYLAND          (Wanted: True): ON
--   VIDEO_WAYLAND_QT_TOUCH (Wanted: ON): ON
--   VIDEO_X11              (Wanted: True): ON
--   VIDEO_X11_XCURSOR      (Wanted: True): ON
--   VIDEO_X11_XINERAMA     (Wanted: True): ON
--   VIDEO_X11_XINPUT       (Wanted: True): ON
--   VIDEO_X11_XRANDR       (Wanted: True): ON
--   VIDEO_X11_XSCRNSAVER   (Wanted: True): ON
--   VIDEO_X11_XSHAPE       (Wanted: True): ON
--   VIDEO_X11_XVM          (Wanted: True): ON
--   WASAPI                 (Wanted: OFF): OFF
--   WAYLAND_LIBDECOR       (Wanted: ON): OFF
--   WAYLAND_SHARED         (Wanted: False): OFF
--   X11_SHARED             (Wanted: ON): ON
--   XINPUT                 (Wanted: OFF): OFF
--
--  CFLAGS:          -m64 -idirafter "/home/stella/.conan/data/sdl/2.0.16/_/_/build/7be1ce258372c81cde134e98e4254a059071ec45/source_subfolder/src/video/khronos" -DHAVE_LINUX_VERSION_H
--  EXTRA_CFLAGS:  -I/usr/include -I/home/stella/.conan/data/openssl/1.1.1k/_/_/package/58aa52996642505eee5f4042cedcf9b4d1c57eb4/include -I/usr/lib/glib-2.0/include -I/usr/include/glib-2.0 -I/usr/include/harfbuzz -I/usr/include/libpng16 -I/usr/include/freetype2 -DHAS_STICKY_DIR_BIT -DHAS_FCHOWN -D_BSD_SOURCE -D_DEFAULT_SOURCE -I/home/stella/.conan/data/libalsa/1.2.4/_/_/package/c9856c71dedf45b7eef47b38ef5a5fca622fe1d1/include -I/home/stella/.conan/data/libcap/2.50/_/_/package/58aa52996642505eee5f4042cedcf9b4d1c57eb4/include -I/home/stella/.conan/data/opus/1.3.1/_/_/package/58aa52996642505eee5f4042cedcf9b4d1c57eb4/include/opus -I/home/stella/.conan/data/opus/1.3.1/_/_/package/58aa52996642505eee5f4042cedcf9b4d1c57eb4/include -DFLAC__NO_DLL -I/home/stella/.conan/data/flac/1.3.3/_/_/package/c0ee7360ba61fdb5568cc4c1bcb0ebd49e4aa293/include -I/home/stella/.conan/data/vorbis/1.3.7/_/_/package/92bf7eef3b44b5628cca3f9dd5c3cf74e6740617/include -I/home/stella/.conan/data/ogg/1.3.4/_/_/package/58aa52996642505eee5f4042cedcf9b4d1c57eb4/include -I/home/stella/.conan/data/libsndfile/1.0.31/_/_/package/6d4901686f6e21208e189e4a8256c909777ff39d/include -D_REENTRANT -I/home/stella/.conan/data/pulseaudio/14.2/_/_/package/1652e103f03ba70f8404bd7d95cb5314c18ac560/include -msse3 -msse2 -msse -m3dnow -mmmx -Wshadow -fvisibility=hidden -Wdeclaration-after-statement -Werror=declaration-after-statement -fno-strict-aliasing -Wall
--  EXTRA_LDFLAGS: -lXau -L/home/stella/.conan/data/wayland/1.19.0/_/_/package/b692d8f5246cdff4da1af4d2015ce9e023b13b6b/lib;-Wl,--no-undefined;-pthread
--  EXTRA_LIBS:    wayland-egl;wayland-cursor;wayland-client;pthread;m;ffi;EGL;xkbcommon;iconv;dl;/home/stella/.conan/data/nas/1.9.4/_/_/package/077884758292d626fd4366e95bb9fe5d8f397c2e/lib/libaudio.a
--
--  Build Shared Library: False
--  Build Static Library: True
--  Build Static Library with Position Independent Code: True
--
-- If something was not detected, although the libraries
-- were installed, then make sure you have set the
-- CFLAGS and LDFLAGS environment variables correctly.
--
-- Configuring incomplete, errors occurred!
See also "/home/stella/.conan/data/sdl/2.0.16/_/_/build/7be1ce258372c81cde134e98e4254a059071ec45/build_subfolder/CMakeFiles/CMakeOutput.log".
See also "/home/stella/.conan/data/sdl/2.0.16/_/_/build/7be1ce258372c81cde134e98e4254a059071ec45/build_subfolder/CMakeFiles/CMakeError.log".
sdl/2.0.16:
sdl/2.0.16: ERROR: Package '7be1ce258372c81cde134e98e4254a059071ec45' build failed
sdl/2.0.16: WARN: Build folder /home/stella/.conan/data/sdl/2.0.16/_/_/build/7be1ce258372c81cde134e98e4254a059071ec45
ERROR: sdl/2.0.16: Error in build() method, line 165
	self._build_cmake()
while calling '_build_cmake', line 245
	cmake = self._configure_cmake()
while calling '_configure_cmake', line 239
	self._cmake.configure(build_dir=self._build_subfolder)
	ConanException: Error 1 while executing cd 'build_subfolder' && cmake -G "Unix Makefiles" -DCMAKE_BUILD_TYPE="Release" -DCONAN_IN_LOCAL_CACHE="ON" -DCONAN_COMPILER="gcc" -DCONAN_COMPILER_VERSION="11.1" -DCONAN_CXX_FLAGS="-m64" -DCONAN_SHARED_LINKER_FLAGS="-m64" -DCONAN_C_FLAGS="-m64" -DBUILD_SHARED_LIBS="OFF" -DCMAKE_INSTALL_PREFIX="/home/stella/.conan/data/sdl/2.0.16/_/_/package/7be1ce258372c81cde134e98e4254a059071ec45" -DCMAKE_INSTALL_BINDIR="bin" -DCMAKE_INSTALL_SBINDIR="bin" -DCMAKE_INSTALL_LIBEXECDIR="bin" -DCMAKE_INSTALL_LIBDIR="lib" -DCMAKE_INSTALL_INCLUDEDIR="include" -DCMAKE_INSTALL_OLDINCLUDEDIR="include" -DCMAKE_INSTALL_DATAROOTDIR="share" -DCONAN_CMAKE_POSITION_INDEPENDENT_CODE="ON" -DCMAKE_EXPORT_NO_PACKAGE_REGISTRY="ON" -DCONAN_EXPORTED="1" -DCONAN_INSTALL_FOLDER="/home/stella/.conan/data/sdl/2.0.16/_/_/build/7be1ce258372c81cde134e98e4254a059071ec45" -DSDL_STATIC_PIC="True" -DSDL_SHARED="False" -DSDL_STATIC="True" -DVIDEO_OPENGL="True" -DVIDEO_OPENGLES="True" -DVIDEO_VULKAN="True" -DSDL_VIDEO_DRIVER_X11_SUPPORTS_GENERIC_EVENTS="1" -DALSA="True" -DHAVE_ASOUNDLIB_H="True" -DHAVE_LIBASOUND="True" -DJACK="False" -DPULSEAUDIO="True" -DSNDIO="False" -DNAS="True" -DNAS_SHARED="False" -DVIDEO_X11="True" -DHAVE_XEXT_H="True" -DVIDEO_X11_XCURSOR="True" -DHAVE_XCURSOR_H="True" -DVIDEO_X11_XINERAMA="True" -DHAVE_XINERAMA_H="True" -DVIDEO_X11_XINPUT="True" -DHAVE_XINPUT_H="True" -DVIDEO_X11_XRANDR="True" -DHAVE_XRANDR_H="True" -DVIDEO_X11_XSCRNSAVER="True" -DHAVE_XSS_H="True" -DVIDEO_X11_XSHAPE="True" -DHAVE_XSHAPE_H="True" -DVIDEO_X11_XVM="True" -DHAVE_XF86VM_H="True" -DVIDEO_WAYLAND="True" -DWAYLAND_SHARED="False" -DWAYLAND_SCANNER_1_15_FOUND="1" -DVIDEO_DIRECTFB="False" -DVIDEO_RPI="False" -DEXTRA_LDFLAGS="-lXau -L/home/stella/.conan/data/wayland/1.19.0/_/_/package/b692d8f5246cdff4da1af4d2015ce9e023b13b6b/lib" -DCMAKE_REQUIRED_INCLUDES="/home/stella/.conan/data/nas/1.9.4/_/_/package/077884758292d626fd4366e95bb9fe5d8f397c2e/include" -Wno-dev '/home/stella/.conan/data/sdl/2.0.16/_/_/build/7be1ce258372c81cde134e98e4254a059071ec45'
```
</details>

--- a/recipes/sdl/all/conanfile.py
+++ b/recipes/sdl/all/conanfile.py
@@ -41,6 +41,7 @@ class SDLConan(ConanFile):
         "opengl": [True, False],
         "opengles": [True, False],
         "vulkan": [True, False],
+        "libunwind": [True, False],
     }
     default_options = {
         "shared": False,
@@ -69,6 +70,7 @@ class SDLConan(ConanFile):
         "opengl": True,
         "opengles": True,
         "vulkan": True,
+        "libunwind": True,
     }
 
     exports_sources = ["CMakeLists.txt", "patches/*"]
@@ -107,6 +109,7 @@ class SDLConan(ConanFile):
             del self.options.wayland
             del self.options.directfb
             del self.options.video_rpi
+            del self.options.libunwind
         if self.settings.os != "Windows":
             del self.options.directx
 
@@ -138,6 +141,8 @@ class SDLConan(ConanFile):
                 self.requires("egl/system")
             if self.options.directfb:
                 raise ConanInvalidConfiguration("Package for 'directfb' is not available (yet)")
+            if self.options.get_safe("libunwind", False):
+                self.requires("libunwind/1.5.0")
 
     def validate(self):
         if self.settings.os == "Macos" and not self.options.iconv:
@@ -233,6 +238,8 @@ class SDLConan(ConanFile):
             elif self.settings.os == "Windows":
                 self._cmake.definitions["DIRECTX"] = self.options.directx
 
+            self._cmake.definitions["HAVE_LIBUNWIND_H"] = self.options.get_safe("libunwind")
+
             # Add extra information collected from the deps
             self._cmake.definitions["EXTRA_LDFLAGS"] = " ".join(cmake_extra_ldflags)
             self._cmake.definitions["CMAKE_REQUIRED_INCLUDES"] = ";".join(cmake_required_includes)
@@ -305,6 +312,8 @@ class SDLConan(ConanFile):
                 self.cpp_info.components["libsdl2"].requires.append("wayland::wayland")
                 self.cpp_info.components["libsdl2"].requires.append("xkbcommon::xkbcommon")
                 self.cpp_info.components["libsdl2"].requires.append("egl::egl")
+            if self.options.get_safe("libunwind"):
+                self.cpp_info.components["libsdl2"].requires.append("libunwind::libunwind")
         elif tools.is_apple_os(self.settings.os):
             self.cpp_info.components["libsdl2"].frameworks = [
                 "CoreVideo", "CoreAudio", "AudioToolbox",

--- a/recipes/sdl/all/conanfile.py
+++ b/recipes/sdl/all/conanfile.py
@@ -312,7 +312,7 @@ class SDLConan(ConanFile):
                 self.cpp_info.components["libsdl2"].requires.append("wayland::wayland")
                 self.cpp_info.components["libsdl2"].requires.append("xkbcommon::xkbcommon")
                 self.cpp_info.components["libsdl2"].requires.append("egl::egl")
-            if self.options.get_safe("libunwind"):
+            if self.options.libunwind:
                 self.cpp_info.components["libsdl2"].requires.append("libunwind::libunwind")
         elif tools.is_apple_os(self.settings.os):
             self.cpp_info.components["libsdl2"].frameworks = [

--- a/recipes/sdl/all/conanfile.py
+++ b/recipes/sdl/all/conanfile.py
@@ -141,7 +141,7 @@ class SDLConan(ConanFile):
                 self.requires("egl/system")
             if self.options.directfb:
                 raise ConanInvalidConfiguration("Package for 'directfb' is not available (yet)")
-            if self.options.get_safe("libunwind", False):
+            if self.options.libunwind:
                 self.requires("libunwind/1.5.0")
 
     def validate(self):
@@ -235,10 +235,10 @@ class SDLConan(ConanFile):
 
                 self._cmake.definitions["VIDEO_DIRECTFB"] = self.options.directfb
                 self._cmake.definitions["VIDEO_RPI"] = self.options.video_rpi
+                self._cmake.definitions["HAVE_LIBUNWIND_H"] = self.options.libunwind
             elif self.settings.os == "Windows":
                 self._cmake.definitions["DIRECTX"] = self.options.directx
 
-            self._cmake.definitions["HAVE_LIBUNWIND_H"] = self.options.get_safe("libunwind")
 
             # Add extra information collected from the deps
             self._cmake.definitions["EXTRA_LDFLAGS"] = " ".join(cmake_extra_ldflags)

